logging/cluster-logging-upgrading.adoc

```diff
@@ -6,14 +6,9 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-.{product-title} version support for Red Hat OpenShift Logging (RHOL)
-[options="header"]
-|====
-|        |4.7          |4.8          |4.9
-|RHOL 5.0|X            |X            |
-|RHOL 5.1|X            |X            |
-|RHOL 5.2|X            |X            |X
-|====
+[id="cluster-logging-supported-versions"]
+== Supported Versions
+include::modules/cluster-logging-supported-versions.adoc[leveloffset=+1]
```

Comment:
This include statement is causing a build failure because this you've deleted that file in this PR.